packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/tests/index.test.js

```diff
@@ -117,4 +117,71 @@ describe('ADMIN | Pages | AUDIT LOGS | ListView', () => {
     await user.click(closeButton);
     expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
   });
+
+  it('should show pagination and be on page 1 on first render', async () => {
+    mockUseQuery.mockReturnValue({
+      data: {
+        results: getBigTestData(15),
+        pagination: {
+          page: 1,
+          pageSize: 10,
+          pageCount: 2,
+          total: 15,
+        },
+      },
+      isLoading: false,
+    });
+
+    const { container } = render(App);
+
+    const pagination = await waitFor(() => container.querySelector('nav[aria-label="pagination"]'));
+
+    expect(pagination).toBeInTheDocument();
+    expect(pagination.querySelectorAll('ul li')[1].querySelector('a')).toHaveTextContent('1');
+    expect(pagination.querySelectorAll('ul li')[1].querySelector('a')).toHaveClass('active');
+  });
```

Comment:
You're running `pagination.querySelectorAll('ul li')` several times, you could save it in a variable instead

But also I think we should try to select items based on their text when possible instead of html tags, to match what a real user would look for when using the app. You can see how we handled it for the [marketplace pagination](https://github.com/strapi/strapi/blob/feature/market-pagination/packages/core/admin/admin/src/pages/MarketplacePage/tests/plugins.test.js#L393)